hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/util/LogCaptor.java

```diff
@@ -114,7 +114,7 @@ private List<String> eventsAt(@NonNull final Level level) {
             }
             // now check if the current match is the level we're looking for
             String matchLevel = m.group(0);
-            if (level.equals(Level.getLevel(matchLevel))) {
+            if (!matchLevel.isEmpty() && level.equals(Level.getLevel(matchLevel))) {
```

Comment:
Needed due to the LOG4J update. `getLevel` now throws in an empty string is passed.